operator/main.go

```diff
@@ -84,7 +85,7 @@ func run() error {
 
 	ctrl.SetLogger(zap.New(zap.UseFlagOptions(&opts)))
 
-	mgr, err := ctrl.NewManager(ctrl.GetConfigOrDie(), ctrl.Options{
```

Comment:
With identical function names it is easy to overlook the package name and use wrong function (the one which doesn't set UserAgent). I suggest to rename `utils.GetConfigOrDie()` to something a bit more specific. I don't have suggestions I would like myself, but a couple would be: `utils.GetConfigWithUserAgentOrDie()` and `utils.GetStackRoxConfigOrDie()`.

Reply:
Changed name to utils.GetRHACSConfigOrDie()